releasenotes/notes/1.0/prepare-1.0-e13b9b8273055775.yaml

```
@@ -0,0 +1,71 @@
+---
+prelude: |
+  We are very pleased to release Qiskit 1.0.0. This release is the
+  culmination of 7 years of development to mature Qiskit into a stable, high-
+  performance SDK for quantum computing, and is the start of a new
+  era for the Qiskit project and community. Besides general performance and
+  stability improvements, the most significant changes with the 1.0.0 
+  release relate to the stability policy, release cycles, and versioning.
+  Qiskit is now formally using `semantic versioning <https://semver.org/>`__ ,
+  and this means that, for the entire lifecycle of the 1.x release series, the project
+  is commited to maintaining backwards compatibility. We're also now starting
```

```suggestion
  Qiskit is now formally using `semantic versioning <https://semver.org/>`__,
  and this means that, for the entire lifecycle of the 1.x release series, the project
  is committed to maintaining backwards compatibility. We're also now starting
```

(symbols after rST links are fine, just not letters)